configure.json

```diff
@@ -1,4 +1,29 @@
 {
+    "mss-check-hasredzone":{
+        "program": "mss-check-redzone",
+        "arguments": { "0": ["0"]},
+        "set-var": { "0": "has-redzone"}
+    },
+    "mss-getlen-redzone_class_var_stack":{
+        "program": "mss-check-redzone",
+        "arguments": { "0": ["1","-vhas-redzone"]},
```

Comment:
Since the simple check-redzone run without argument, this one can start with 0, complying with all other test cases.
Also, there is no need to add `-vhas-redzone`. Instead, add a requirement for `mss-check-redzone`